spec/support/webmock/court_data_api/webmock.rb

```diff
@@ -277,6 +277,16 @@
     )
   end
 
+  config.before(:each, :stub_cdapi_v2_hearing_summary2) do
```

Comment:
Is there a better name we can give this tag (and the json file) than just `2`? How is this different from hearing_summary_response 1? What is different about it that we need to sometimes use one and sometimes the other?